Added logic for handling 'about:blank' to download()

hbregalad opened the pull request:
It could be placed somewhere else perhaps, if you want, but this is where made sense to me.

Basically I've been dealing with some pages that are peppered with iframes that start out pointing to 'about:blank' and it's breaking things.

Andereoo commented:
Thanks for this contribution! Do you happen to have the url for one of those page so I could test this on my end?

Andereoo commented:
@hbregalad I'm not too sure why this was closed, but I just wanted to let you know that I have updated TkinterWeb to add support for `about:blank` pages. I modified your code a bit to add more general support for `about:` pages (making it easier to add new ones if needed). Simply update TkinterWeb and your issues with `about:blank` frames should be resolved. As always, feel free to let me know if you have any other suggestions or if you notice any bugs!

hbregalad commented:
Thanks!

On Tue, Apr 12, 2022, 14:02 Andereoo ***@***.***> wrote:

> @hbregalad <https://github.com/hbregalad> I'm not too sure why this was
> closed, but I just wanted to let you know that I have updated TkinterWeb to
> add support for about:blank pages. I modified your code a bit to add more
> general support for about: pages (making it easier to add new ones if
> needed). Simply update TkinterWeb and your issues with about:blank frames
> should be resolved. As always, feel free to let me know if you have any
> other suggestions or if you notice any bugs!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Andereoo/TkinterWeb/pull/34#issuecomment-1097030083>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACDO23GXUNFEYKGTNR6AZXDVEW3EVANCNFSM5SUG4RUA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
